sources/install/package_crypto.sh

```diff
@@ -40,6 +40,15 @@ function install_rsacracker() {
     add-to-list "RsaCracker,https://github.com/skyf0l/RsaCracker,Powerful RSA cracker for CTFs. Supports RSA - X509 - OPENSSH in PEM and DER formats."
 }
 
+function install_sage {
+    # CODE-CHECK-WHITELIST=add-aliases
+    colorecho "Installing Sage"
+    fapt sagemath
+    add-history sage
+    add-test-command "sage --help"
+    add-to-list "sage,https://www.sagemath.org,SageMath is a free open-source mathematics software system licensed under the GPL."
+}
+
```

Comment:
Can you move your installation in a `install_crypto_apt_tools` function ? It's faster for builds when there are other tools via apt.

Like this one : https://github.com/ThePorgs/Exegol-images/blob/dev/sources/install/package_ad.sh#L6

Reply:
@LighTend3r